<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>message_recieve</class>
 <widget class="QWidget" name="message_recieve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>966</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/search</normaloff>:/search</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>0</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>个人中心</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/person</normaloff>:/person</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>小白盒</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/gamer</normaloff>:/gamer</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>106</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>首页</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/First</normaloff>:/First</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>热点</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/hot</normaloff>:/hot</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="text">
       <string>关注</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/focus</normaloff>:/focus</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>511</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>暂无评论</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>没赞</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>没关注</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="all_comment">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    /* 静态无边框 */
    border: none;
    background-color: transparent;
    
    /* 文字样式 */
    color: #666666;
    font-weight: 500;
    font-family: &quot;Microsoft YaHei&quot;, sans-serif;
    
    /* 鼠标光标 */
    cursor: pointer;
    
    /* 底部透明边框（为下划线预留位置） */
    border-bottom: 3px solid transparent;
}
QPushButton[selected1=&quot;true&quot;] {
    color: rgb(0,0,0);  
    font-weight: 600;
    border-bottom: 3px solid rgb(0,0,0);  
}</string>
      </property>
      <property name="text">
       <string>所有评论</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="prize">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    /* 静态无边框 */
    border: none;
    background-color: transparent;
    
    /* 文字样式 */
    color: #666666;
    font-weight: 500;
    font-family: &quot;Microsoft YaHei&quot;, sans-serif;
    
    /* 鼠标光标 */
    cursor: pointer;
    
    /* 底部透明边框（为下划线预留位置） */
    border-bottom: 3px solid transparent;
}
QPushButton[selected2=&quot;true&quot;] {
    color: rgb(0,0,0);  
    font-weight: 600;
    border-bottom: 3px solid rgb(0,0,0);  
}</string>
      </property>
      <property name="text">
       <string>赞</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="focus">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    /* 静态无边框 */
    border: none;
    background-color: transparent;
    
    /* 文字样式 */
    color: #666666;
    font-weight: 500;
    font-family: &quot;Microsoft YaHei&quot;, sans-serif;
    
    /* 鼠标光标 */
    cursor: pointer;
    
    /* 底部透明边框（为下划线预留位置） */
    border-bottom: 3px solid transparent;
}
QPushButton[selected3=&quot;true&quot;] {
    color: rgb(0,0,0);  
    font-weight: 600;
    border-bottom: 3px solid rgb(0,0,0);  
}</string>
      </property>
      <property name="text">
       <string>关注</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
